### DISCUSSION TITLE
Open/closed folder icons

### DISCUSSION THREAD

**Discussion by hdbv95** [twilio-labs/paste#3295] [Request: Icon] [answered]
### Description

It would be nice to have these UI icons. the use case would be on a sidebar 

<img width="273" alt="Screenshot 2023-06-26 at 17 16 30" src="https://github.com/twilio-labs/paste/assets/28691512/79d16276-5e02-46d5-a231-97f1f442bc9e">

>note: It also would be nice once the sidebar component is done to allow the user to select the icon 
### Link to icon

https://www.streamlinehq.com/icons/streamline-light/files-folders/folders/folder
https://www.streamlinehq.com/icons/streamline-light/files-folders/folders/folder-open

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by libertycarlin** [comment, marked as answer]
hello @hdbv95! 

Our team just created a UI kit to unify how navigation looks and feels across Twilio products. With that being said, we do not recommended using icons within the side nav, as we just solved for a cohesive navigational experience. 

The navigational components will be released as early as next week! It includes purely presentational components as building blocks, and are crafted to accommodate all existing features across Console, Segment, Flex, Aion, and SendGrid.

Hope this helps!

Cheers,
Liberty
